tests/test_coreg/test_blockwise.py

```diff
@@ -153,3 +146,40 @@ def test_blockwise_coreg_pipeline(self, blockwise_coreg, example_data, tmp_path)
 
         valid = (expected.data.data != expected.nodata) & (aligned.data.data != aligned.nodata)
         assert np.allclose(expected.data.data[valid], aligned.data.data[valid], atol=20)
+
+    def test_ransac_on_horizontal_tiles(self, blockwise_coreg) -> None:
+        """Test case where RANSAC works on horizontal tiles."""
+        x = np.linspace(0, 100, 50)
+        y = np.full_like(x, 50)
```

Comment:
Not sure this is testing 1D: Both `x` and `y` are of size 50? I think one of them should be of size 1.
Same for vertical below.